src/components/plots/hierarchical-tree-genes/HierarchicalTreeGenes.jsx

```diff
@@ -12,17 +12,15 @@ const HierarchicalTreeGenes = (props) => {
 
   const onDrop = (info) => {
     const {
-      dragNode, dropPosition, dropToGap,
+      dragNode, node, dropPosition,
     } = info;
 
     // if dropped in place, ignore
     // dragNode.key is str, dropPosition is int
     if (dragNode.key == dropPosition) return;
 
-    // If not dropped in gap, ignore
-    if (!dropToGap) return;
-
-    let newPosition = dropPosition - (dragNode.key < dropPosition ? 1 : 0);
+    // if dropping below the initial position subtract 1, if dropping to secondary position add 1
+    let newPosition = dropPosition - (dragNode.key < dropPosition ? 1 : 0) + node.dragOver;
```

Comment:
Don't forget to refactor this if you decided to refactor the code in `HierarchicalTree`